reana/reana_dev/release.py

```diff
@@ -202,4 +190,55 @@ def release_pypi(ctx, component, timeout):  # noqa: D301
         click.secho(f"{component} successfully released on PyPI", fg="green")
 
 
+@click.option("--user", "-u", default="reanahub", help="DockerHub user name [reanahub]")
+@release_commands.command(name="release-helm")
+@click.pass_context
+def release_helm(ctx, user):  # noqa: D301
+    """Release REANA as a Helm chart."""
+    component = "reana"
+    version = get_current_component_version_from_source_files(component)
+    is_chart_releaser_installed = which("cr")
+    github_pages_branch = "gh-pages"
+    package_path = ".cr-release-packages"
+    index_path = ".cr-index"
+    repository = f"https://{user}.github.io/{component}"
+
+    is_component_releasable(component, exit_code=True, display=True)
+    if not is_chart_releaser_installed:
+        click.secho(
+            "Please install chart-releaser to be able to do a Helm release", fg="red",
+        )
+        sys.exit(1)
+
+    if not os.getenv("CR_TOKEN"):
+        click.secho(
+            "Please provide your GitHub token as CR_TOKEN environment variable",
+            fg="red",
+        )
+        sys.exit(1)
+
+    for cmd in [
+        f"rm -rf {package_path}",
+        f"mkdir {package_path}",
+        f"rm -rf {index_path}",
+        f"mkdir {index_path}",
+        f"helm package helm/reana --destination {package_path} --dependency-update",
+        f"cr upload -o {user} -r {component}",
```

Comment:
Discovered while amending https://github.com/reanahub/reana/pull/411 and checking out https://github.com/diegodelemos/reana/releases/tag/reana-0.7.0-alpha.2:

We should pass `--release-name-template "{{ .Version }}"` so the release on GitHub doesn't contain the `reana-` prefix (more [here](https://github.com/helm/chart-releaser#create-github-releases-from-helm-chart-packages)).